ert3/data/__init__.py

```diff
@@ -1,19 +0,0 @@
-from ert3.data._record import Record
```

Comment:
This makes us loose the history of the file. Not very critical for this particular file, but if you're able to make it into a move instead that would be awesome. Maybe not feasible due to the extent of the changes from `ert3` -> `ert`?

Reply:
I did the move again, but for some reason it does not recorderd for the `__init__.py`, which is strange, because it does it nicely for the `_record.py`, which is logged as having been moved.

Reply:
I think it happens because the diff is too substantial and it will look good if you move it in one commit and change it in the next, but that makes the repo broken in the first commit. Which is not worth it given the content of the file.

Let us keep it as is 👍🏻